apps/web/src/lang/fr.json

```diff
@@ -5402,6 +5406,10 @@
     "defaultMessage": "À la prochaine fois!",
     "description": "Title for the page users land on after successful logout."
   },
+  "caSdGd": {
+    "defaultMessage": "Notes",
+    "description": "Title displayed on the Pool Candidates table notes column."
+  },
```

Comment:
Could you maybe please use `adminMessages.notes` or some other recycling
We've already got "Notes" quite a bit already 🤣 

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/041464cb-456f-4c60-9644-d9838ffb7a23)

Reply:
Yeah, good suggestion! @mnigh Is this a safe string to move to `adminMessages`?

Reply:
It already is in there but not used that many times I think

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/9fc248d1-67c7-4567-bd7f-def1ed1d93f1)

Reply:
For one word-ers, I think that description field's usefulness is debatable

Reply:
Oops, did not notice that. Updated in [update to use common notes message](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7215/commits/268d4ef9433850ebef8ec7b5a398a0fb0628f101)